run-tests.py

```diff
@@ -38,7 +38,6 @@
     'server/pulp/server/agent/',
     'server/pulp/server/async/',
     'server/pulp/server/auth/',
-    'server/pulp/server/common/',
```

Comment:
good catch :fishing_pole_and_fish: 

I meant to comment on that in the PR and forgot. `server/pulp/server/common` only had one file, `openssl.py`. It got moved to `repoauth/pulp/repoauth` which is already covered, so the dir was removed.